usb_winusb.go

```diff
@@ -67,7 +67,7 @@ func usbopen(cid string, timeout time.Duration, serial string) (err error) {
 	}
 
 	var timeoutMs int64 = timeout.Milliseconds()
-	if (timeoutMs > math.MaxUint32) {
+	if timeoutMs > math.MaxUint32 {
```

Comment:
Was this for style matching? In general I'd like to see us moving towards all conditional expressions in ()'s..